db-client/src/main/java/com/splicemachine/db/client/am/PreparedStatement.java

```diff
@@ -941,8 +944,8 @@ public void setRowId(int parameterIndex, java.sql.RowId x) throws SQLException{
                     agent_.logWriter_.traceEntry(this, "setRowId", parameterIndex, x);
                 }
 
-                final int paramType =
-                        getColumnMetaDataX().getColumnType(parameterIndex);
+                getColumnMetaDataX().getColumnType(parameterIndex);
```

Comment:
this call is not assign to any variable, usually paramType. Looks like not needed any more and can be removed.

Reply:
I would leave this like this since `getColumnType` can throw. We never used `paramType`, but if we want to keep the current behaviour, then we should continue calling this function.